src/App.tsx

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
-import './App.css';
-import { getNumbers } from './utils';
+import React, { useState } from "react";
+import "./App.css";
+import { getNumbers } from "./utils";
+import { Pagination } from "./components/Pagination";
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const total = 42;
+
+const items = getNumbers(1, total).map((n) => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [perPage, setPerPage] = useState<number>(5);
+
+  const START_ITEM = perPage * currentPage - perPage + 1;
+  const END_ITEM =
+    perPage * currentPage > items.length ? items.length : perPage * currentPage;
```

Comment:
```suggestion
  const END_ITEM = perPage * currentPage > items.length
    ? items.length 
    : perPage * currentPage;
```
to fix the lint error - FYI